src/Queue/JobDetail.js

```diff
@@ -16,39 +66,149 @@ function JobDetail(props) {
       }
     }
     // const resultUrl = `https://raw.githubusercontent.com/jonathanduperrier/nmpi-job-manager-app-reactjs/master/db_${id}.json`;
-    const resultUrl = `https://nmpi.hbpneuromorphic.eu/api/v2/results/${id}/?collab_id=neuromorphic-testing-private`;
+    const resultUrl = `https://nmpi.hbpneuromorphic.eu/api/v2/results/${id}`;
```

Comment:
todo: move API base URL (`https://nmpi.hbpneuromorphic.eu/api/v2/`) to a globals.js file